packages/lib-utils/src/k8s/hooks/watch-resource-types.ts

```diff
@@ -7,13 +7,13 @@ import type {
 } from '../../types/k8s';
 
 export type WatchK8sResult<R extends K8sResourceCommon | K8sResourceCommon[]> = [
-  data: R,
+  data: R | unknown,
```

Comment:
I would also like to revisit these `|`'s. We really should be returning predictable types on these hooks.

Reply:
I don't believe these are required for the build to pass. I'll get rid of them.